ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -208,7 +219,31 @@ export default defineComponent({
       })
     })
 
+    const firstNameRule = customRules(
+      required('Enter a first name'),
+      isStringOrNumber(),
+      maxLength(15),
+      invalidSpaces()
+    )
+
+    const middleNameRule = customRules(isStringOrNumber(), maxLength(15), invalidSpaces())
+
+    const lastNameRule = customRules(
+      required('Enter a last name'),
+      isStringOrNumber(),
+      maxLength(25),
+      invalidSpaces())
+
+    const businessNameRule = customRules(required('Business name is required'), invalidSpaces())
```

Comment:
No `maxLength` rules for these fields below?

Reply:
added `maxLength` to it.